InnerEye/ML/deep_learning_config.py

```diff
@@ -609,7 +611,7 @@ def get_train_epochs(self) -> List[int]:
         Returns the epochs for which training will be performed.
         :return:
         """
-        return list(range(self.start_epoch + 1, self.num_epochs + 1))
+        return list(range(self._start_epoch + 1, self.num_epochs + 1))
```

Comment:
I'd be tempted to kill this "get_train_epochs" thing. It's main use was to speed up training in Windows, by reducing the number of process spawns.

Reply:
OK, I can see you found a clean way of doing that, so let's just leave it in place.